Source/Cells/Popover/FORMPopoverFieldCell.m

```diff
@@ -137,12 +126,23 @@ - (void)titleLabelPressed:(FORMFieldValueLabel *)titleLabel {
 
     [self updateContentViewController:self.contentViewController withField:self.field];
 
-    if (!self.popoverController.isPopoverVisible) {
-        [self.popoverController presentPopoverFromRect:self.bounds
-                                                inView:self
-                              permittedArrowDirections:UIPopoverArrowDirectionUp | UIPopoverArrowDirectionDown
-                                              animated:YES];
+    self.contentViewController.modalPresentationStyle = UIModalPresentationPopover;
+    UIPopoverPresentationController *presentationController = [self.contentViewController popoverPresentationController];
+    presentationController.sourceView = self;
+    presentationController.sourceRect = self.bounds;
+
+    UIViewController *topViewController = [self topViewController];
+    [topViewController presentViewController:self.contentViewController animated:YES completion:nil];
+}
+
+- (UIViewController *) topViewController {
```

Comment:
Looks good, I just tried on the simulator :clap: 

Would you mind moving this method to a category on `UIViewController`?